### ISSUE TITLE
api  errors

### ISSUE THREAD

**Issue by tzapu** [corpetty/go-alethio-api#3]
this really really needs to return errors instead of an empty struct,  when errors are returned from the API.

**Comment by corpetty** [comment]
yeah I chose this just to get started, this should certainly relay the given error.
